policies/Sponsorship_Policy.md

```diff
@@ -0,0 +1,154 @@
+
+
+### Levels of Sponsorship provided to OpenSSL
+
+Details of Sponsorship options are located on OpenSSL's [Sponsorship webpage].
+
+
+### Sponsorship Decisions
+
+Only the OMC or Authorised Approver can, in line with The Policy, decide whether to accept sponsorship. Only the OMC, in line with The Policy, can approve the provision of sponsorship.
+
+
+Sponsorship provided via [GitHub Sponsors] in part or in whole, is deemed to be
+accepted sponsorship by the OMC, however the OMC reserves the right to reject 
+any sponsorship provided via the [GitHub Sponsors] where it is deemed by the 
+OMC, to be inappropriate.
+
+
+### Sponsorship assessment
+
+#### OpenSSL Receiving Sponsorship
+
+The OMC or Authorised Approver, will assess each offer to provide sponsorship, except where provided via [GitHub Sponsors], under the following criteria:
+* whether it aligns with OpenSSL's Mission and values;
+* whether sponsorship supports the achievement of OpenSSL's goals, objectives 
+and priorities;
+* whether it does not cause a conflict of interest, or regulatory or legal 
+infringement;
+* whether it complies with OpenSSL's Code of Conduct Policy;
+* whether it damages OpenSSL's reputation.
+
+Where a sponsorship is reported to the OMC as a potential inappropriate sponsor
+the above criteria will be used to assess that sponsorship regardless of where
+the sponsorship resides, i.e. [GitHub Sponsors] or direct arrangement with 
+OpenSSL.
+
+
+**Recording Sponsorship of OpenSSL**
+
+* For Sponsorship made via [GitHub Sponsors], <$400/month and one-off payments,  
+the information provided via [GitHub Sponsors] registration process is deemed 
+sufficient record of the sponsorship provided.
+* For Sponsorship at Platinum Level or above, a Sponsorship Agreement, should 
+be entered into. It should clearly set out:
+   * the rights and obligations between both parties;
+   * clearly confirm any commitments made to each other;
+   * include Term of Sponsorship, any exclusivity requirements, payment 
+requirements, agreed benefits, and termination of agreement details.
+* For all Sponsorship, at Bronze Level or above, (including GitHub Sponsors 
+$400/month or above) the sponsorship must be documented and recorded in an 
+appropriate system so that it can be identified and managed.
+
+Details to be recorded in this system are:
+     * Name of organisation, individual or entity providing sponsorship;
+     * Duration of sponsorship arrangement, including start and end dates (where applicable);
+     * Value of sponsorship in $US and local currency (where applicable);
+     * Details of what is being supplied under the sponsorship e.g. financial aid, 
+resources etc;
+     * Any conditions that apply to the sponsorship;
+     * Whether permission to display logos, company details and links to their 
+websites has been provided;
+     * Contact details for the organisation, Individual or Entity.
```

Comment:
This part doesn't seem to render as a list, probably because of the indentation before the \*s

Reply:
I think I've fixed it, it's showing as a list when I look at the rendered version, but let me know if it shows differently for you.  (markdown sometimes has a mind of it's own! ;-) )

Reply:
It renders much better now